tests/.infra/crw-ci/dashboard-next/pr-check/Jenkinsfile

```diff
@@ -260,7 +215,8 @@ pipeline {
 spec:
   server:
     cheImage: '${cheImageRepo}'
-    cheImageTag: '${mutableCheImageTag}'
+    cheImageTag: '${cheImageTag}'
```

Comment:
do we need to specify the cheImage or cheImageTag ?

Reply:
cheImage is cheImageRepo
cheImageTag is cheImageTag 
=)
What do you mean?